data/txHandler.go

```diff
@@ -137,7 +146,14 @@ func (handler *TxHandler) backlogWorker() {
 				logging.Base().Debugf("could not remember tx: %v", err)
 				continue
 			}
-			handler.net.Relay(handler.ctx, protocol.TxnTag, wi.rawmsg.Data, false, wi.rawmsg.Sender)
+
+			reencoded, err := reencode(wi.rawmsg.Data)
```

Comment:
Is this the same as `protocol.Encode(verifiedTxGroup)`? If so could we could do that and avoid decoding twice?

Reply:
it should be, yes! fixed, thanks